app/views/spree/admin/shared/_order_tabs.html.haml

```diff
@@ -1,5 +1,8 @@
 - content_for :page_title do
-  = t(:order)
+  - if @order.cart?
+    = t(:new_order)
+  - else
+    = t(:order)
```

Comment:
This is not correct. Order workflow cart? cant be used here.
Do you understand the approach? The point of extracting page_title partial was to extract/delete it from this partial.
These lines should be deleted from this partial.

Reply:
There was some weirdness without this in there, so I will step back as well as implement your other suggestions and see what I get. I'll let you know if I have any questions. Thanks again!